src/app/features/cases/components/organisms/CaseDetail/CaseDetailSection.tsx

```diff
@@ -79,11 +78,13 @@ const CaseDetailSection: FC<Props> = ({ id, dataSource, title, data, footer, isB
           }) }
         </Grid>
       </SectionRow>
-      { showFooter &&
-      <SectionRow>
-        <CenteredAnchor href={ footer!.link }>{ footer!.title }</CenteredAnchor>
-      </SectionRow>
-      }
+      { showFooter && (
+        <SectionRow>
+          <CenteredAnchor href={ footer!.link } target="_blank" rel="noopener noreferrer">
```

Comment:
Dit klinkt alsof er twee componenten in één zit. Ik zou hier een 'Center' Component verwachten met daarin een 'Anchor' component.

Reply:
Dat is het ook een beetje. Een StyledLink die vervolgens weer met styling in het midden(centre) wordt gezet. Een wrapper is misschien meer op zijn plaats. Wat denk jij? @jmassink

Reply:
StyledAnchor is het geworden... 🥳